src/app/services/impl/StompBrokerService.ts

```diff
@@ -21,30 +22,48 @@ export class StompBrokerService extends BrokerService {
     if (!this.stompClient.active) {
       this.stompClient.configure(this.stompConfig);
       this.stompClient.activate();
-      this.connect$.next();
+      this.stompClient.connectionState$.subscribe((state: RxStompState) => {
+        console.log(`RxStomp's state: ${RxStompState[state]}`);
```

Comment:
Is this a debug message or a useful message you want to keep in console?

Reply:
We would like to keep this since we are going to troubleshoot over a ton of WebSockets stuff